### ISSUE TITLE
[patch] win_service.stop() fails

### ISSUE THREAD

**Issue by gurubert** [saltstack/salt#44272]
### Description of Issue/Question
The method stop() in the state module win_service fails if the service has already been stopped.

### Versions Report
Salt Version:
           Salt: 2017.7.2

This patch solves the issue:
```
--- win_service.py.orig	2017-10-04 18:02:10.000000000 +0200
+++ win_service.py	2017-10-25 14:46:21.439718502 +0200
@@ -444,8 +444,9 @@
     try:
         win32serviceutil.StopService(name)
     except pywintypes.error as exc:
-        raise CommandExecutionError(
-            'Failed To Stop {0}: {1}'.format(name, exc[2]))
+        if exc[0] != 1062:
+            raise CommandExecutionError(
+                'Failed To Stop {0}: {1}'.format(name, exc[2]))
 
     attempts = 0
     while info(name)['Status'] in ['Running', 'Stop Pending'] \
```

**Comment by gtmanfred** [comment]
Would you mind submitting this as a PR?

https://docs.saltstack.com/en/latest/topics/development/contributing.html

Thanks,
Daniel

**Comment by rallytime** [comment]
Closed via #44295
